src/organisms/__tests__/MaterialPage.test.js

```diff
@@ -8,8 +8,32 @@ import {
   getByText
 } from "@testing-library/react";
 
-import { mockMaterialQuery } from "./mock_data/mockQueries";
-//jest.requireMock to require a mock module?
+const mocksQuery = [
```

Comment:
Should be a different PR, but it's not the end of the world, will be fine to add.